docs/xtrabackup-option-reference.md

```diff
@@ -171,7 +179,10 @@ that will read the data and pipe it to 2 compression threads.
 
 ### --compress-zstd-level(=#)
 
-This option specifies `ZSTD` compression level. The default value is 1. Allowed range of values is from 1 to 19. 
+This option specifies `ZSTD` compression level. Compression levels provide a trade-off between the compression speed and the compressed files' size. A lower compression level provides faster compression speed but larger file sizes. A higher compression level provides lower compression speed but smaller file sizes. For example, set level 1 if the compression speed is the most important for you. Set level 19 if the compressed file size is the most important.
```

Comment:
remove the single quote after the files. You can write it as compressed file size.

Reply:
Fixed